pyqtgraph/graphicsItems/PlotItem/PlotItem.py

```diff
@@ -1002,10 +1002,18 @@ def setClipToView(self, clip):
         
     def clipToViewMode(self):
         return self.ctrl.clipToViewCheck.isChecked()
-
+    
+    
+    def _max_trace_wrapper_slot(self, check_state):
```

Comment:
I would rename this to be more verb-y, e.g. `_handle_max_traces_toggle`, but that's a style choice for which we don't have a guideline.